src/System.Runtime.Extensions/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,12 @@
+kernel32.dll!ExpandEnvironmentStringsW
```

Comment:
For all these additional PInvokes what logic are you using to determine if they are allowed or not? It would seem as though they should be promoted to the win32 allow list if the are truly allowed. I would hope only exceptional cases would ever be in these files. Things like clrcompression or sni or things we don't want to support on OneCore/UWP.